.github/workflows/scripts/json-file-to-job-output.js

```diff
@@ -0,0 +1,27 @@
+module.exports = async ({ core, filePath }) => {
+    try {
+        const fs = require('fs');
+        const content = await readFile(fs, filePath);
```

Comment:
**Suggestion:** maybe use [`fs.readFileSync()`](https://nodejs.org/api/fs.html#fsreadfilesyncpath-options) instead?

Although it's blocking, I think it shouldn't be too much of an issue here, on the other hand it would make the script simpler. What do you think?